oggm/utils.py

```diff
@@ -3161,10 +3161,10 @@ def shape_factor_huss(widths, heights, is_rectangular):
 
     # Ensure bool (for masking)
     is_rectangular = is_rectangular.astype(bool)
-
     shape_factors = np.ones(widths.shape)
```

Comment:
always repeating initialization could be substituted by storing shape factor array in the flowmodel and passing the reference to the shape factor functions